public/CREDITS.TXT

```diff
@@ -0,0 +1,99 @@
+Lanea Zimmerman (AKA Sharm)

```

Comment:
open game art can produce an automated collection of credits from everything you put in a collection which makes for compliance with gpl, cc-by etc. That what you used here? Otherwise might be good for the future. Some licenses also require credits to be visible to end user, don't remember which ones though...

Reply:
Noice that credits are included though! Kudos!